### PR TITLE
[201911] [Flex Counters] add CLI for PG drop packets counters (counterpoll, show/clear counters)

### PR DESCRIPTION
**Pull request by ayurkiv-nvda**
Should be merged after https://github.com/Azure/sonic-swss/pull/2263

Appropriate PR in master:https://github.com/Azure/sonic-utilities/pull/1355, https://github.com/Azure/sonic-utilities/pull/1461, https://github.com/Azure/sonic-utilities/pull/1583

#### What I did
Added new option for "counterpoll" utility
Added new CLI commands to view and clear PG dropped packet statistics.
Added the new CLI commands to the command reference guide.

#### How I did it
Need to merge PG drop functionality to 201911 

#### How to verify it
```
admin@arc-switch1041:~$ counterpoll pg-drop enable  --> to enable the new counter
admin@arc-switch1041:~$ counterpoll show  --> check new INGRESS_PG_STAT_DROP counter status
```

Check counters
```
admin@arc-switch1041:~$ redis-cli -n 2
127.0.0.1:6379[2]> HGETALL COUNTERS:oid:0x1a000000000062
1) "SAI_INGRESS_PRIORITY_GROUP_STAT_XOFF_ROOM_WATERMARK_BYTES"
2) "0"
3) "SAI_INGRESS_PRIORITY_GROUP_STAT_SHARED_WATERMARK_BYTES"
4) "0"
5) "SAI_INGRESS_PRIORITY_GROUP_STAT_DROPPED_PACKETS"
6) "0"
```

**show priority-group drop counters**
```
Ingress PG dropped packets:
     Port    PG0    PG1    PG2    PG3    PG4    PG5    PG6    PG7
---------  -----  -----  -----  -----  -----  -----  -----  -----
Ethernet0    800    801    802    803    804    805    806    807
Ethernet4    400    401    402    403    404    405    406    407
Ethernet8    100    101    102    103    104    105    106    107
...
```
**sonic-clear priority-group drop counters**
`Cleared PG drop counters`

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@neethajohn following your ask enclose in the backport PR. Appreciate if you can prioritized the review.

File: doc/Command-Reference.md [hunk 1]
```diff
@@ -4528,7 +4543,7 @@ This command displays the user persistet-watermark for the queues (Egress shared
   admin@sonic:~$ show queue persistent-watermark multicast
   ```
 
-- NOTE: Both "user watermark" and "persistent watermark" can be cleared by user:
+- NOTE: "user watermark", "persistent watermark" and "ingress dropped packets" can be cleared by user:
```
Comment (neethajohn) [inline doc/Command-Reference.md:4546]: please also include the command to clear the pg drop counters

Comment (ayurkiv-nvda) [reply doc/Command-Reference.md:4546]: @neethajohn  done

**Comment by ayurkiv-nvda** [comment]
> This pull request **introduces 1 alert** and **fixes 1** when merging [31130ef](https://github.com/Azure/sonic-utilities/commit/31130efa05b12cef0b87338380171e5d3520e973) into [913df4e](https://github.com/Azure/sonic-utilities/commit/913df4e2faf6e70e0aebb01d81f79694a6d8ee20) - [view on LGTM.com](https://lgtm.com/projects/g/Azure/sonic-utilities/rev/pr-7b7d0f08f63c560ae891859e9f35783a9617379f)
> 
> **new alerts:**
> 
> * 1 for Unused import
> 
> **fixed alerts:**
> 
> * 1 for Unused import

Implicit usage of **import mock_tables.dbconnector**

**Review by neethajohn — APPROVED** [review]

**Comment by ayurkiv-nvda** [comment]
Merged few additional fixes :
https://github.com/Azure/sonic-utilities/pull/1678
https://github.com/Azure/sonic-utilities/pull/1679 **NOTE**: I didn't managed to backport UT for this PR, faced with some  infrastructure problem for 201911 sonic-utulities UT. Meybe need to invest more time 
@neethajohn please review existing backport

**Comment by ayurkiv-nvda** [comment]
@neethajohn gentle reminder regarding additional fix for this PR https://github.com/Azure/sonic-utilities/commit/84fe4da828783af59c0f9b4adc60a10194b0be87 
Can you confirm that it ok?

**Review by neethajohn — APPROVED** [review]

**Pull request merged**